po/zh_CN.po

```diff
@@ -41,6 +28,20 @@ msgstr ""
 # token: 记号
 # universal (variable): 通用
 # valid & invalid: 有效 & 无效
+msgid ""
+msgstr ""
+"Project-Id-Version: fish 1.21.8\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-03-10 16:38+0100\n"
+"PO-Revision-Date: 2025-09-30 19:14+0800\n"
+"Last-Translator: Yuyi Wang <Strawberry_Str@hotmail.com>\n"
+"Language-Team: \n"
+"Language: zh_CN\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.7\n"
+
 msgid ""
 "\n"
```

Comment:
LGTM.
BTW another quirk is that the `# fish-section-*` comments show up in `poedit` as comments on individual messages.
I don't know any better, it was a quick hack